app/views/collections/_form.html.erb

```diff
@@ -16,6 +16,7 @@
         ] %>
       <%= form.input :active %>
       <%= form.button :submit, class: 'btn-primary' %>
+      <%= link_to 'Cancel', collections_path, html_options={style: "margin-left: 25px"} %>
```

Comment:
This doesn't seem like valid Ruby.  Do we have a test for this file?

Reply:
Test added.

Reply:
I don't understand why you are assigning a hash to the local variable named `html_options`

Reply:
It's part of the method signature: https://apidock.com/rails/ActionView/Helpers/UrlHelper/link_to

Reply:
But when you call a method you don't pass in the method signature.  In a signature that means, the third parameter is called html_options, and if it is not provided, here is the default.  When you're invoking you don't want the `html_options=`

Reply:
Removed `html_options`.